tests/test_net_adatpter.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 - 2021 MONAI Consortium
```

Comment:
there's a typo in the filename. could you make the test cases smaller? it's currently slow on CPU

Reply:
Sure, that may because the newly added 3D test cases are too big, I changed the input image size in the latest commit.

Thanks.